bench/random.go

```diff
@@ -55,3 +68,15 @@ func (b *RandomSetBits) Run(ctx context.Context) *Result {
 	}
 	return results
 }
+
+var letters = "abcdefghijklmnopqrstuvwxyz"
+
+func (b *RandomSetBits) getAttrQuery(rng *rand.Rand, rowID int64) string {
+	if b.NumAttrs <= 0 || b.NumAttrValues <= 0 {
+		return ""
+	}
+	attri := rng.Intn(b.NumAttrs)
+	val := rng.Intn(b.NumAttrValues)
+	return fmt.Sprintf("SetRowAttrs(rowID=%d, frame='%s', %c%d=%d)", rowID, b.Frame, letters[attri%len(letters)], attri, val)
```

Comment:
Doesn't this produce only "a0", "b1", "c2", etc? Why not just use "aXX"?

Reply:
until it wraps around "a26", "b27", etc. In any case, it certainly isn't fully exercising attributes and we'll probably need something more flexible at some point (maybe in a different benchmark). I'd be fine with simplifying this if you thinks it's too confusing.

Reply:
nah, this is fine